lib/fittings/swagger_render.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = function create(fittingDef, bagpipes) {
```

Comment:
'bagpipes' is defined but never used.<br>'fittingDef' is defined but never used.